docs/contributing-analyzer.md

```diff
@@ -24,19 +24,13 @@ Visual Studio 2022 version 17.6+ is required to build the project (due to source
     - .NET Framework 4.8 SDK
     - .NET Framework 4.8 Targeting pack
     - .NET Framework 4.7.2 Targeting pack
-    - .NET Framework 4.6 Targeting pack
     - .NET SDK
     - .NET Compiler Platform SDK
 1. Install also:
-    - [.NET Core 2.1 SDK](https://dotnet.microsoft.com/en-us/download/dotnet)
```

Comment:
Was this just obsolete?
The other PR fixed the .NET Framework projects, I don't think I saw .NET Core 2.1 anywhere.

Reply:
Yes, obsolete.